apps/mull-ui/src/app/app.tsx

```diff
@@ -21,6 +31,8 @@ export const App = () => {
           {<Redirect to={ROUTES.HOME} />}
         </Route>
         <Route exact path={ROUTES.CREATE_EVENT} component={CreateEventPage} />
+        <Route exact path={ROUTES.LOGIN} component={LoginPage} />
+        <Route exact path={ROUTES.REGISTER} component={RegisterPage} />
```

Comment:
Yeah, I think that for authentication to show routes should be done when the #30 is done. We will leave the showNavigation function for now but will replace it when the backend is done.

Reply:
As mentioned in #30, we will implement the authentication and this will hide the pages if you're not authenticated. For now, we'll leave it as it is for now